crates/tabby-common/src/usage.rs

```diff
@@ -16,6 +16,26 @@ struct UsageTracker {
 
 impl UsageTracker {
     fn new() -> Self {
+        if std::env::var("TABBY_DISABLE_USAGE_COLLECTION").is_ok() {
+            eprintln!(
+                "
+  \x1b[34;1mTELEMETRY\x1b[0m
+
+  TABBY_DISABLE_USAGE_COLLECTION is set and telemetry has been disabled.
+
+  \x1b[1mWelcome to Tabby!\x1b[0m
+
+  If you have any questions or would like to engage with the Tabby team, please join us on Slack
+  (https://tinyurl.com/35sv9kz2).
+
+"
+            );
+            return Self {
+                id: String::from("00000000-0000-0000-000000000000"),
```

Comment:
Make the field optional, instead of setting a dummy value.

Reply:
:+1: